Pipfile

```diff
@@ -47,6 +47,8 @@ black = "*"
 pytest-mock = "*"
 pytest-mypy = "*"
 moto = "*"
+ipython = "*"
+prompt-toolkit = "==2.0.10"
```

Comment:
why would that be another dependency?

Reply:
https://github.com/ipython/ipython/issues/11975